analyzer/src/funTest/assets/projects/synthetic/pip-expected-output.yml

```diff
@@ -99,25 +97,24 @@ packages:
 - id: "PyPI::itsdangerous:0.24"
   purl: "pkg:pypi/itsdangerous@0.24"
   authors:
-  - "Armin Ronacher"
+  - "Armin Ronacher <armin.ronacher@active-4.com>"
   declared_licenses:
-  - "BSD License"
+  - "UNKNOWN"
   declared_licenses_processed:
     unmapped:
-    - "BSD License"
-  description: "Various helpers to pass trusted data to untrusted environments and\
-    \ back."
+    - "UNKNOWN"
+  description: "It's Dangerous"
   homepage_url: "http://github.com/mitsuhiko/itsdangerous"
   binary_artifact:
-    url: "https://files.pythonhosted.org/packages/dc/b4/a60bcdba945c00f6d608d8975131ab3f25b22f2bcfe1dab221165194b2d4/itsdangerous-0.24.tar.gz"
+    url: ""
```

Comment:
Why's this missing?

Reply:
I guess because PyPI only lists the .tar.gz URL for this package: https://pypi.org/pypi/itsdangerous/0.24/json
I will collect such issues and report them to python-inspector once I'm done with this PR and the upgrade to 0.8.2 which was released in the meantime.

Reply:
> we used the `.tar.gz` as the source artifact

You mean "binary artifact" here, I guess?